tweetapi/login.py

```diff
@@ -78,6 +81,9 @@ def get(self,request):
         return response
 
 class ChangePassword(CreateAPIView,TwitterUserPermission):
+    authentication_classes = [Authentication]
+    permission_classes = [IsAuthenticated]
+    permission_classes = [TwitterUserPermission]
     serializer_class = ChangeSerializers
```

Comment:
why plural?

Reply:
i dont unserstand what this mean sir, but i guess is because i had two permission_classes when i can put them in a single array, but is fixed now sir

Reply:
serializer name